m4/with_libical.m4

```diff
@@ -69,5 +69,10 @@ AC_DEFUN([PBS_AC_WITH_LIBICAL],
   AC_SUBST(libical_inc)
   AC_SUBST(libical_lib)
   AC_DEFINE([LIBICAL], [], [Defined when libical is available])
+  PKG_CHECK_MODULES([libical_new_api],
+    [libical >= 2],
+    AC_DEFINE([LIBICAL_NEW_API], [], [Defined when libical version >= 2]),
```

Comment:
I think LIBICAL_NEW_API is a bit too generic.  If libical3 came out with a new API, we'd have a variable saying we were using the "new" api, but in fact we were only using ical2's API.  Maybe change this to LIBICAL_API2.